<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purBox</class>
 <widget class="QDialog" name="purBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Purchase Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
   QDialog {
       background-color: #34495e;
   }
   QLabel {
       color: white;
       font-family: "Segoe UI";
   }
   QLineEdit {
       background-color: white;
       border-radius: 5px;
       padding: 5px;
       font-size: 15px;
   }
   QPushButton {
       background-color: #27ae60;
       color: white;
       font-size: 18px;
       border-radius: 10px;
       padding: 8px 16px;
   }
   QPushButton:hover {
       background-color: #1e8449;
   }
   </string>
  </property>

  <!-- Title -->
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>ðŸ›’ Purchase Window</string>
   </property>
  </widget>

  <!-- Book Name -->
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>ðŸ“– Enter Book Name</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
  </widget>

  <!-- Quantity -->
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>ðŸ”¢ Enter Book Quantity</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
  </widget>

  <!-- OK Button -->
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>130</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string> OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
